lib/settings/pages/general_settings_page.dart

```diff
@@ -359,6 +365,13 @@ class _GeneralSettingsPageState extends State<GeneralSettingsPage> with SingleTi
                           iconDisabled: Icons.person_off_rounded,
                           onToggle: (bool value) => setPreferences(LocalSettings.useDisplayNamesForUsers, value),
                         ),
+                        ToggleOption(
+                          description: 'Disable All Score Counters',
+                          value: disableScoreCounters,
+                          iconEnabled: Icons.score_rounded,
+                          iconDisabled: Icons.score_rounded,
+                          onToggle: (bool value) => setPreferences('setting_disable_score_counters', value),
```

Comment:
I'm not sure if this will solve your issue, but even if not, this should be 
``` dart
setPreferences(LocalSettings.disableScoreCounters, value),
```

Reply:
This does fix it, thank you! I hate chasing down errors with these settings switches. Luckily each time I do I learn something new.